doc/crypto/api/ops/signature.rst

```diff
@@ -681,6 +744,65 @@ Asymmetric signature functions
     ..  note::
         To perform a multi-part hash-and-sign signature verification algorithm, first use a :ref:`multi-part hash operation <hash-mp>` to hash the message and then pass the resulting hash to `psa_verify_hash()`. :code:`PSA_ALG_GET_HASH(alg)` can be used to determine the hash algorithm to use.
 
+
+.. function:: psa_verify_message_with_context
+
+    .. summary::
+        Verify the signature of a message with a public key and a non-default context. For hash-and-sign algorithms, this includes the hashing step.
```

Comment:
Do we want to refer to a 'non-default' context? Should we say 'non-empty' context instead - or can we anticipate that a default would ever be different from an empty string?